framework/resource/retryresource/crud_resource_ops_wrapper_test.go

```diff
@@ -0,0 +1,364 @@
+package retryresource
+
+// TODO Uncomment tests when new framework.Resource interface is created.
```

Comment:
Commented tests. I don't want to deal with them before getting approval. Things may change.